launcher/minecraft/skins/SkinModel.cpp

```diff
@@ -23,10 +23,17 @@
 #include "FileSystem.h"
 #include "Json.h"
 
-static QImage improveSkin(const QImage& skin)
+static QImage improveSkin(const QImage& _skin)
 {
+    auto skin = _skin;
+    // It seems some older skins may use this format, which can't be drawn onto
+    // https://github.com/PrismLauncher/PrismLauncher/issues/4032
+    // https://doc.qt.io/qt-6/qpainter.html#begin
+    if (skin.format() != QImage::Format_RGBA8888) {
+        skin = skin.convertToFormat(QImage::Format_RGBA8888);
```

Comment:
```suggestion
        skin = skin.convertToFormat(QImage::Format_RGB32);
```
Any reason to convert to this format specifically? According to [the docs](https://doc.qt.io/qt-6/qpainter.html#performance), this will have "significantly worse" performance when painting